src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -480,7 +480,7 @@ public function __toString()
         try {
             $content = $this->getContent();
         } catch (\LogicException $e) {
-            trigger_error($e->getMessage(), E_USER_ERROR);
+            return trigger_error($e, E_USER_ERROR);
```

Comment:
trigger_error expects a string as first parameter and not an exception?! Also this changes the return value of the method. The method is supposed to return a string and not a boolean from trigger_error.